webapp/package.json

```diff
@@ -20,8 +20,7 @@
 		"cypress:run:firefox": "cypress run --browser firefox",
 		"cypress:run:edge": "cypress run --browser edge",
 		"cypress:run:electron": "cypress run --browser electron",
-		"cypress:open": "cypress open",
-		"updatesnapshots": "jest --updateSnapshot"
```

Comment:
I wonder why this was deleted.

Reply:
This line is redundant with `updatesnapshot` on line 11 so I removed it.